### ISSUE TITLE
Empty response (improve error handling and responses)

### ISSUE THREAD

**Issue by Kamoba** [pollinations/pollinations#816]
While testing I found a prompt returning empty string.
```
https://text.pollinations.ai/sexy%20teacher%20in%20classroom
```
😊
But no error, is returned!
I know to generating image it's recommended to use image.pollinations.ai but... just reporting issue.

**Comment by flowa-ai** [comment]
if I had to guess this might have something to do with nsfw prompt detection or caching issue. although I am not 100% sure why it failed to generate.

if you would like faster help without using github you can also join the pollinations server on discord:
https://discord.com/invite/k9F7SyTgqn

**Comment by Kamoba** [comment]
Actually for now there is no nsfw, they may enforce it later. Because if you change a bit it works like replacing teacher by professor or girl. I do not need it at all, just was wondering why any error code was not sent, look to me an error in model.

**Comment by flowa-ai** [comment]
maybe a caching error? 

these work: 
https://text.pollinations.ai/sexy%20teacher%20in%20a%20classroom
https://text.pollinations.ai/a%20sexy%20teacher%20in%20classroom

**Comment by Kamoba** [comment]
Yes that's what I'm saying you added 'a' and it works. Caching in server? Maybe if failed for some reason then cached and keep sending empty string... 
Anyway a good model/App should instead send something like "_Hummn, I didn't get that. Please try again_."

**Comment by voodoohop** [comment]
We need to improve our error handling. Will turn this into a more specific issue later
